README.md

```diff
@@ -1,29 +1,38 @@
 # Neutron Cross-Chain Vault Deployment
 
-A production-ready cross-chain Bitcoin vault system built on the Valence Protocol that enables users to deposit WBTC on Ethereum while generating yield on Neutron through Mars Protocol lending and Supervaults.
+A production-ready cross-chain vault system built on the Valence Protocol that enables users to deposit selected denoms on Ethereum while generating yield on Neutron through Mars Protocol lending and Supervaults.
 
 ## Architecture
 
 The system operates across three blockchain networks:
 - **Ethereum**: User-facing ERC-4626 vault where users deposit collateral
-- **Neutron**: CosmWasm-based liquidity provision via Mars Protocol + Supervaults  
+- **Neutron**: CosmWasm-based liquidity provision via Mars Protocol + Supervaults
 - **Cosmos Hub**: IBC/ICA bridging/messaging between Ethereum and Neutron
 - **IBC Eureka**: Ethereum ↔ Cosmos Hub bridging
+- **ZK Co-processor**: Generating proofs for IBC Eureka route queries and withdrawal obligations
 
 ## Components
```

Comment:
We need a section here on Vault Types (e.g., LBTC, LBTC-USDC), and with each we link to the directory. Additionally, we should a 1-line description in this top level repository. This comment is non blocking for merging this PR or initial customer handoff. We can add this as a follow on issue

Reply:
We should also have note about Phase shifts. We can say something like "Highlighted vaults above allow the Vault Owner to trigger a phase shift. This phase shift changes the yield generating policy by forcing the strategist to deploy liquidity to a different venue."